lib/mysql.js

```diff
@@ -444,6 +444,12 @@ MySQL.prototype.fromColumnValue = function(prop, val) {
           lat: val.y,
         };
         break;
+      case 'ObjectID':
```

Comment:
Interesting. Why do we need to deal with `ObjectID`, considering that it's a type specific to MongoDB? Is this related to changes made in juggler v4?

Reply:
@bajtos See tests in https://github.com/strongloop/loopback-connector-mysql/blob/1849507e065985ea0242440979a073bcebe4682e/test/mysql.test.js#L14, I guess it's for consistent behavior across all connectors.

Reply:
I see. `test/mysql.test.js` is not part of the shared test suite imported from juggler, so why are we fixing it in this pull request? Can you please open a new pull requests to fix problems that are unrelated to the shared test suite first?

Reply:
@bajtos Interesting 🤔 I thought you have noticed that after configuring the shared tests, the connector tests run with juggler v4(which this PR does) instead of juggler v3(the current master), and that's how all these fixes happen. 
And I thought "running connector tests with juggler v4" is what we intend to do...